include/os/windows/zfs/sys/wzvol.h

```diff
@@ -219,13 +220,19 @@ typedef enum {
 	ActionUnmap
 } MpWkRtnAction;
 
+
 typedef struct _MP_WorkRtnParms {
 	pHW_HBA_EXT		pHBAExt;
 	PSCSI_REQUEST_BLOCK	pSrb;
 	PEPROCESS		pReqProcess;
 	MpWkRtnAction		Action;
 	ULONG			SecondsToDelay;
-	CHAR			pQueueWorkItem[1];
+	PVOID				 pUio;
```

Comment:
Whats `pUio` for? I'm not seeing anything using it in this PR, is being used in future code?

Reply:
This was used in ZFSin earlier when uio was allocated on heap. It is not used now.